docs/docsite/rst/shared_snippets/installing_collections_git_repo.txt

```diff
@@ -0,0 +1,84 @@
+You can install a collection in a git repository by providing the URI to the repository instead of a collection name or path to a ``tar.gz`` file. The collection must contain a ``galaxy.yml`` file, which will be used to generate the would-be collection artifact data from the directory. The URI should be prefixed with ``git+`` (or with ``git@`` to use a private repository with ssh authentication) and optionally supports a comma separated version (for example, a commit or tag).
```

Comment:
URL and URI seem to be used inconsistently across the docs.

P.S. Nitpick: it's better to split paragraphs into multiple shorter lines in order to make the rst source more readable.

Reply:
Roles support URLs. Collections support URIs.

Reply:
I think I saw mentions of URLs below in this document

Reply:
Fixed